napari/_qt/menus/file_menu.py

```diff
@@ -173,3 +179,10 @@ def _add_sample(*args, plg=plugin_name, smp=samp_name):
 
                 menu.addAction(action)
                 action.triggered.connect(_add_sample)
+
+    def update(self):
+        for ax in self.actions():
+            data = ax.data()
+            if data:
+                enabled_func = data.get('enabled', lambda: True)
```

Comment:
... and then you can do 
```suggestion
                enabled_func = data.get('enabled', lambda event: True)
```